### ISSUE TITLE
Is it possible to add chat history to extractor workflow?

### ISSUE THREAD

**Issue by vnguye65** [neuml/txtai#575]
I have the following config.yml. Is it possible to add chat history to the template? 
```
writable: true
path: index

embeddings:
  path: sentence-transformers/multi-qa-mpnet-base-dot-v1
  content: true

extractor:
  path: google/flan-t5-base
  context: 3
  output: flatten

workflow:       
  search:
    tasks:
      - task: extractor
        template: |          
          Chat history: 
          {history}
          Answer the following question using only the context below and the above chat history. Say "I don't have data on that" when the question can't be answered.
          Question: 
          {text}
          Context: 
        
        action: extractor

```

**Comment by davidmezzetti** [comment]
I'll have to take a look and try it out. I'm assuming you tried this and it didn't work as expected?

**Comment by davidmezzetti** [comment]
I just checked in a change that should make this possible. You can install txtai from GitHub to give it a try.

**Comment by davidmezzetti** [comment]
Will be released in 6.2.0
